bears/js/JSONFormatBear.py

```diff
@@ -10,10 +10,7 @@
 
 
 class JSONFormatBear(LocalBear):
-    try:
-        DecodeError = json.decoder.JSONDecodeError
-    except AttributeError:
-        DecodeError = ValueError
+    DecodeError = json.decoder.JSONDecodeError
```

Comment:
Ahh. Then you can't put the pytest --cov-fail-under condition :| because we do test for multiple branches. in fact we have some windows specific branches which are collated from the coverage on windows...